Runtime/NotchSolutionUtility.cs

```diff
@@ -21,7 +21,7 @@ public static Rect SimulateSafeAreaRelative
             get
             {
                 var rectString = EditorPrefs.GetString(simulateSafeAreaRectKey, "0,0,1,1");
```

Comment:
I think you forgot that line
```suggestion
                var rectString = EditorPrefs.GetString(simulateSafeAreaRectKey, "0;0;1;1");
```